app/src/main/java/com/codesoom/assignment/service/ProductService.java

```diff
@@ -0,0 +1,52 @@
+package com.codesoom.assignment.service;
+
+import com.codesoom.assignment.ProductNotFoundException;
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.repository.ProductRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+@Transactional
+public class ProductService {
+    private final ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    public List<Product> getProducts() {
+        return productRepository.findAll();
+    }
+
+    public Product getProduct(Long id) {
+        Product product = productRepository.findById(id).orElseThrow(() -> {
```

Comment:
한 줄에 `.` 이 하나씩만 오게 개행해보세요.
```suggestion
        Product product = productRepository.findById(id)
            .orElseThrow(() -> {
```

Reply:
네 감사합니다. 자동으로 해주는 거 찾아봐야겠네요ㅜㅜ